control/controlclient/direct.go

```diff
@@ -310,11 +310,16 @@ type loginOpt struct {
 	Logout bool
 }
 
+type httpClient interface {
```

Comment:
I guess if you didn't use a RoundTripper that means you meant for this to be http.Client-like and follow redirects?

Reply:
added a comment, it's not meant to stop redirects (which should not be happening anyways, maybe that belongs in a different layer though). The goal with this is to just have a single interface that can be used between noise and nacl